corona_analysis/scrapers/rmf24.py

```diff
@@ -48,28 +50,38 @@ def scrape(data_url: str = URL) -> dict:
         remove_suffix(remove_prefix(remove_prefix(i, "[["), "Date.UTC"), "]]")
         for i in deaths
     ]
-    recovers = [
+    recoveries = [
         remove_suffix(remove_prefix(remove_prefix(i, "[["), "Date.UTC"), "]]")
-        for i in recovers
+        for i in recoveries
+    ]
+    vaccinations = [
+        remove_suffix(remove_prefix(remove_prefix(i, "[["), "Date.UTC"), "]]")
+        for i in vaccinations
     ]
 
     sick = [i.split("),") for i in sick]
     deaths = [i.split("),") for i in deaths]
-    recovers = [i.split("),") for i in recovers]
+    recoveries = [i.split("),") for i in recoveries]
+    vaccinations = [i.split("),") for i in vaccinations]
 
     sick = [["-".join(i[0][1:].split(",")), i[1]] for i in sick]
     deaths = [["-".join(i[0][1:].split(",")), i[1]] for i in deaths]
-    recovers = [["-".join(i[0][1:].split(",")), i[1]] for i in recovers]
+    vaccinations = [["-".join(i[0][1:].split(",")), i[1]] for i in vaccinations]
 
-    return {"sick": sick, "deaths": deaths, "recovers": recovers}
+    return {
+        "sick": sick,
+        "deaths": deaths,
+        "recovers": recoveries,
+        "vaccinations": vaccinations,
+    }
 
 
-def get_data(sick: list, deaths: list, recovers: list) -> dict:
+def get_data(
+    sick: List[int], deaths: List[int], recoveries: List[int], vaccinations: List[int]
+) -> Dict[str, pd.DataFrame]:
```

Comment:
To stay consistent across all scrapers `get_data` should either have no parameters or only URL parameter.